### PR TITLE
New header

### PR DESCRIPTION
**Pull request by Shine-neko**

### CODE REVIEW

File: src/Tempo/Bundle/AppBundle/Event/MenuBuilderEvent.php [hunk 1]
```diff
@@ -17,8 +17,9 @@
 
 class MenuBuilderEvent extends Event
 {
-    const FRONTEND_MAIN = 'tempo.menu.builder.frontend.main';
-    const FRONTEND_USER = 'tempo.menu.builder.frontend.user';
+    const MAIN = 'tempo.menu.builder.main';
```
Comment (amenophis) [inline src/Tempo/Bundle/AppBundle/Event/MenuBuilderEvent.php]: align values ?

Comment (Shine-neko) [inline src/Tempo/Bundle/AppBundle/Event/MenuBuilderEvent.php]: why ? :D

Comment (amenophis) [inline src/Tempo/Bundle/AppBundle/Event/MenuBuilderEvent.php]: readibility ? :D

Comment (Shine-neko) [inline src/Tempo/Bundle/AppBundle/Event/MenuBuilderEvent.php]: Unnecessary. php-cs no longer does

**Pull request merged**